project-addons/product_stock_unsafety/product.py

```diff
@@ -79,7 +79,8 @@ def calc_joking_index(self):
                 filter_ids.append(stock_product_id.id)
         avg = joking_tot / len(filter_ids)
         for product in product_obj.search([]):
-            if product.type != 'product' or product.id not in filter_ids:
+            if product.type != 'product' or product.id not in product_ids \
+                    or  (product.product_brand_id and product.product_brand_id.not_compute_joking or False):
```

Comment:
Esto déjalo como estaba por defecto en el merge ya lo hice yo ayer.

`if product.type != 'product' or product.id not in filter_ids:`